core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
-        classifier_model_instance = cls(
-            id=instance_id, exp_id=exp_id,
+        classifier_data_model_instance = cls(
+            id=instance_id, job_request_id=job_request_id, exp_id=exp_id,
             exp_version_when_created=exp_version_when_created,
             state_name=state_name, algorithm_id=algorithm_id,
             cached_classifier_data=cached_classifier_data,
```

Comment:
Here and everywhere else, should not this be `classifier_data` (according to [doc](https://docs.google.com/document/d/1kACmivEvZVlTYBdqxy-mmdTi3sg_NbkZdS10T-ya-pg/edit#heading=h.uvfzqc5tabwr))?